public/locales/en/translation.json

```diff
@@ -124,5 +124,9 @@
         ]
       }
     ]
+  },
+  "aboutUsPage": {
+    "header": "About Us",
+    "infoAboutUs": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor i reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
```

Comment:
since 128 already has aboutUs in its word you dont need to say it again in the key on 130. aboutUsPage.infoAboutUs -> see the redundancy

aboutUsPage.header
aboutUsPage.info (i would probably call this bodyText)